api/firewaller/relation.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package firewaller
+
+import (
+	"gopkg.in/juju/names.v2"
+
+	"github.com/juju/juju/apiserver/params"
+)
+
+// Relation represents a juju relation as seen by the firewaller worker.
+type Relation struct {
+	st   *State
```

Comment:
It aligns with the other entities in that package. Although Relation doesn't need it at the moment so I've removed it until (or if) it is required.